src/cc/CCupgrades.h

```diff
@@ -21,38 +21,70 @@
 #include <map>
 #include <vector>
 
-const int32_t CCASSETS_OPDROP_FIX_TOKEL_HEIGHT = 286359;  // 26 Nov + 90 days
-const int32_t CCASSETS_OPDROP_FIX_TKLTEST_HEIGHT = 243159;  // 26 Nov + 60 days
+#include "version.h"
+#include "komodo_defs.h"
 
 namespace CCUpgrades  {
 
+    // asset chain activation heights
+    const int64_t CCASSETS_OPDROP_FIX_TOKEL_HEIGHT = 286359;  // 26 Nov + 90 days
+    const int64_t CCASSETS_OPDROP_FIX_TKLTEST_HEIGHT = 243159;  // 26 Nov + 60 days
+
+    const int64_t CCMIXEDMODE_SUBVER_1_TKLTEST_HEIGHT = 100000000;  // TBD
+    const int64_t CCMIXEDMODE_SUBVER_1_DIMXY24_HEIGHT = 100000000;  // TBD
+    const int64_t CCMIXEDMODE_SUBVER_1_DIMXY28_HEIGHT = 100000000;  // TBD
+    const int64_t CCMIXEDMODE_SUBVER_1_DIMXY32_HEIGHT = 100000000;  // TBD
+    const int64_t CCMIXEDMODE_SUBVER_1_TKLTEST2_HEIGHT = 89940;  // approx 17 May 2022 10:00a.m. UTC
+    const int64_t CCMIXEDMODE_SUBVER_1_TOKEL_TIMESTAMP = nS6Timestamp; // bound to annual NN S6 update
+    const int64_t CCMIXEDMODE_SUBVER_1_DIMXY33_TIMESTAMP = 1653064202; // test HF 20 May 2022
+
+    // latest protocol version:
+    const int     CCMIXEDMODE_SUBVER_1_PROTOCOL_VERSION = 170010; 
```

Comment:
it actually will be bumped on Jun 24 from 170009 to CCMIXEDMODE_SUBVER_1_PROTOCOL_VERSION=170010,
since then i plan to bump it each time when consensus is changes